spark/README.md

```diff
@@ -0,0 +1,26 @@
+# Apache Spark Helm Chart
+
+Apache Spark is a fast and general-purpose cluster computing system including Apache Zeppelin.
+
+* http://spark.apache.org/
+* https://zeppelin.apache.org/
+
+Inspired from Helm Classic chart https://github.com/helm/charts
+
+Generate using `helm create spark`
```

Comment:
Should this be generate*d*? Not sure if this is relevant to mention though, given all Charts should be generated this way.